core/lib/object_store/src/gcs.rs

```diff
@@ -205,6 +205,14 @@ impl ObjectStore for GoogleCloudStorage {
     async fn remove_raw(&self, bucket: Bucket, key: &str) -> Result<(), ObjectStoreError> {
         self.remove_inner(bucket.as_str(), key).await
     }
+
+    fn get_storage_prefix_raw(&self, bucket: Bucket) -> String {
```

Comment:
Bikeshedding: `get_` prefix [isn't usually](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter) used for getters in Rust (at least, in the standard library); so this method can be called `storage_prefix_raw`. (`get_raw` is different IMO, because it performs a non-trivial action, and there would be nothing left if "get" is removed.)

Reply:
good to know, corrected